### PR TITLE
adds Nullable annotation to Session methods' return type

### PR DESCRIPTION
**Pull request by phiSgr**
Motivation:

Kotlin has nullable types, but it needs annotation from Java code to know if something is nullable.

Modifications:

Return types in the Session API are annotated @Nullable.
Comments in the Kotlin sample use the types as they are called in Kotlin.

Result:

Kotlin compiler now knows the nullability of values.

### CODE REVIEW

**Comment by phiSgr** [comment]
In Kotlin, there's nullable (`?` suffix), not-null (no suffix), and "platform type" (`!` suffix in IDE).

Without the `Nonnull` or `Nullable` annotations, a Java function's return type get resolved to the platform type.
They can be used as if they're not null, or can be safe-called (`?.methodCall`).

When a platform type value is written to a not-null field, `kotlin.jvm.internal.Intrinsics.checkNotNull` is inserted.
So the old code `val myPojo: MyPojo = session.get("key")` will throw if it's null.

File: gatling-core-java/src/main/java/io/gatling/javaapi/core/Session.java [hunk 1]
```diff
@@ -43,6 +44,7 @@ public Session(io.gatling.core.session.Session wrapped) {
    * @param <T> the type of the desired value
    * @return the value if it exists, null otherwise
    */
+  @Nullable
```
Comment (slandelle) [inline gatling-core-java/src/main/java/io/gatling/javaapi/core/Session.java:47]: For consistency:
```java
public <T> @Nullable T get(@Nonnull String key)
```

**Review by slandelle — CHANGES_REQUESTED** [review]
Thanks a lot!
Just one small nit.

File: gatling-core-java/src/main/java/io/gatling/javaapi/core/Session.java (hunk 1 above)
Comment (phiSgr) [reply gatling-core-java/src/main/java/io/gatling/javaapi/core/Session.java]: Interesting. That was what I wrote in the first place, but IntelliJ warned "'Nullable' not applicable to type use".
It compiles fine apparently.

**Review by slandelle — APPROVED** [review]

**Comment by slandelle** [comment]
Cherry-picked. Thanks!

**Pull request closed without merging**